cate_resources/cate.py

```diff
@@ -0,0 +1,82 @@
+import re
+from credentials import USERNAME, PASSWORD, CURRENT_CLASS
+from cate_resources.scraper_init import InitialisationScrapeCient
+import requests
+from bs4 import BeautifulSoup
+import bs4.element
+from datetime import datetime
+from functools import reduce
+
+
+class CateScraperClient(InitialisationScrapeCient):
+    def __init__(self):
+        super().__init__()
+
+    """function to change the self.modules_all_specs dictionary, for each module in the current period and year a list 
+    of urls to link and download using download_spec(self, spec_url) method"""
+
+    def get_all_specs(self, table_complete):
```

Comment:
for all get methods, should have a return statement, in this case since this method is updating the values, you can call it 
```suggestion
    def set_all_specs(self, table_complete):
```